doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -254,6 +250,10 @@ curly braces.
 
 The second, more powerful control on printing is by using :n:`@syntax_modifier`\s. Here is an example
 
+.. coqtop:: in
+
+   Definition IF_then_else (P Q R:Prop) := P /\ Q \/ ~ P /\ R.
```

Comment:
This is the only change that's strictly needed for backporting (the "not truly recursive fixpoint" is in a coqtop::in so not displayed)